pytest_html/plugin.py

```diff
@@ -551,30 +583,12 @@ def generate_summary_item(self):
         return unicode_doc.decode("utf-8")
 
     def _generate_environment(self, config):
```

Comment:
You can replace this entire function and where it is called just do: `self.EnvironmentTable(config).environment_table`

Reply:
Ok will be done in next version